src/commands/ref_command_handler.js

```diff
@@ -34,22 +34,50 @@ function cppReference(message, args) {
                 }
             }
 
-
             const results = [];
             for (const ref of hrefs) {
                 if (ref.search(args[0]) > 0) {
                     console.log(ref);
                     results.push("https://en.cppreference.com/" + ref);
                 }
             }
+			
             if (results.length > 0) {
-                channel.send(results);
+                var errorMessage = "There are too many results, please use a more specific query";
```

Comment:
Doesn't really need to be defined here, this can be put directly into the embed

Reply:
This is because I was originally checking ``2000 - errorMessage.length``, to be able to put just the top results into an embed. Turns out every embed field has a max of 1024 characters, so I dropped it. That's just the leftover. Probably I could still do that using multiple fields, but oh well, didn't think of that before ;) Do you think I should put that too? Or is it unecessary?